### ISSUE TITLE
Libtidy 5.6.0 issue

### ISSUE THREAD

**Issue by nono303** [OSTC/php-sdk-binary-tools#47]
Hi,
Since **libtidy 5.6.0** update (2018-10-17) there’s an error compiling `--with-tidy`
`fatal error LNK1120: 171 external symbols unresolved`
We must compile tidy ext shared to work `--with-tidy=shared`
Seems to be an error linked with php-src:
`/ext/tidy/php_tidy.def` included in `/ext/tidy/config.w32` on:
```
if (!PHP_TIDY_SHARED) {
	ADD_DEF_FILE("ext\\tidy\\php_tidy.def");
}
```
But maybe it’ll be better to keep **libtidy 5.4.0** until php-src are updated

**Comment by weltling** [comment]
Thanks for the report. This kind of issue is not about the binary SDK but the PHP core, so should have been reported on bugs.php.net. Anyway - ext/tidy is compiled shared in the standard build, so the case with the static compilation is probably just not synced. The 5.6.0 release had some security fixes, so it were preferable to use the never dependency. Otherwise this issue of course needs to be fixed in the core. Actually I'm even not sure the libtidy symbols need to be exported at all, lets see.

Thanks.

**Comment by nono303** [comment]
Hi @weltling
Thanks’ for your feedback!
Yes, it’s not an issue linked to binary SDK but to the the PHP core. I just opened it to trace the actual issue.

> Actually, I'm even not sure the libtidy symbols need to be exported at all

I think you’re wright because even with 5.4 static configuration there warning about duplicate symbol exports...

**Comment by weltling** [comment]
This should have been fixed in the core now, for 7.2+.

Thanks.
